tests/integration/test_distributed_async_insert_for_node_changes/configs/remote_servers.xml

```diff
@@ -14,5 +14,21 @@
                 </replica>
             </shard>
         </test_cluster>
+        <test_replica_cluster>
+            <shard>
+                <internal_replication>true</internal_replication>
+                <replica>
+                    <host>node1</host>
+                    <port>9000</port>
+                </replica>
```

Comment:
https://github.com/zhongyuankai/ClickHouse/blob/optimize_cluster_node_data/tests/integration/test_distributed_async_insert_for_node_changes/test.py#L87-#L97
It will be dynamically increased to two `replica` here, it should be able to cover `_all_replicas`.